tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -76,6 +76,11 @@ trait WebDav {
 
 	private $oldDavSlowdownSetting = null;
 
+	/**
+	 * @var int
+	 */
+	private $newDavSlowdownSettingSeconds = 0;
```

Comment:
```suggestion
	private $oldDavSlowdownSettingSeconds = 0;
```
I believe the name refers to the old version of the DAV endpoint

Reply:
I don't know if `oldDavSlowdownSetting` (line 77) is meaning that it is the `previousDavSlowdownSetting` that was in effect before the scenario started, or if it is a setting that only applies to the "old" DAV endpoint. The actual config key is `dav.slowdown`, which does not mention which of the DAV endpoints it refers to.

"Naming is hard"

Reply:
I was wrong, its called `old` because it's used in `resetOldSettingsAfterScenario` to reset settings after the scenario